src/main/java/picard/sam/markduplicates/UmiAwareDuplicateSetIterator.java

```diff
@@ -59,7 +62,9 @@
     private boolean isOpen = false;
     private UmiMetrics metrics;
     private boolean haveWeSeenFirstRead = false;
-    private double expectedCollisions = 0;
+    private Histogram<String> observedUmis = new Histogram<>();
```

Comment:
these should be part of the metric, not the iterator. ..unless you need them in the iterator.